### ISSUE TITLE
Add Mean Pairwise Squared Error loss function

### ISSUE THREAD

**Issue by annxingyuan** [tensorflow/tfjs#829]
Copied from original issue: https://github.com/tensorflow/tfjs/issues/131

**Comment by gaikwadrahul8** [comment]
Hi, @annxingyuan 

Apologize for the delayed response and I see we have updated our TFJS API official documentation w.r.t Training-Losses [here](https://js.tensorflow.org/api/latest/#Training-Losses) so May I know that's what you were looking to add or have I missed something here please let me know ? Thank you!
